templates/default/docker_daemon.yaml.erb

```diff
@@ -0,0 +1,28 @@
+# Generated by Chef, local modifications will be overwritten
+
+init_config:
```

Comment:
@kurochan Thanks! Could you possibly try to replace this section with a similar one to the `instances` one? This retains flexibility in the Chef template if/when the Agent config changes.

Something like:
```ruby
<%= JSON.parse(({ 'init_config' => @ init_config }).to_json).to_yaml %>
```